### PR TITLE
fix(bazel): ensure jsr250 is on the classpath for JDK9+ java_proto_toolchain extraction

### PR DESCRIPTION
**Pull request by jaysachs**

### CODE REVIEW

File: kythe/release/release.BUILD [hunk 1]
```diff
@@ -25,15 +25,21 @@ construct_vnames_config(
 # cross-language metadata file generation.
 proto_lang_toolchain(
     name = "java_proto_toolchain",
-    command_line = "--java_out=annotate_code,shared,immutable,no_enforce_api_compatibility:$(OUT)",
```
Comment (schroederc) [inline kythe/release/release.BUILD]: Why this change?

File: kythe/release/release.BUILD [hunk 2]
```diff
@@ -25,15 +25,21 @@ construct_vnames_config(
...
+    command_line = "--java_out=annotate_code,shared,immutable:$(OUT)",
     runtime = ":protobuf",
 )
 
+java_import(
+	name = "jsr250",
+	jars = [ "jsr250-api-1.0.jar" ],
```
Comment (schroederc) [inline kythe/release/release.BUILD]: This file does not exist in our release archive.

**Comment by shahms** [comment]
Jay, is this still relevant?
